bolt-google-cloud-functions-example/src/main/java/functions/SlackOAuthFunction.java

```diff
@@ -0,0 +1,73 @@
+package functions;
+
+import com.google.cloud.functions.HttpFunction;
+import com.google.cloud.functions.HttpRequest;
+import com.google.cloud.functions.HttpResponse;
+import com.slack.api.bolt.App;
+import com.slack.api.bolt.request.Request;
+import com.slack.api.bolt.request.RequestHeaders;
+import com.slack.api.bolt.response.Response;
+import com.slack.api.bolt.util.SlackRequestParser;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+
+import static java.util.stream.Collectors.joining;
+
+/**
+ * An HttpFunction that handles incoming requests from Slack.
+ */
+public class SlackOAuthFunction implements HttpFunction {
```

Comment:
We will remove this copied base class once we release the updated SlackApiFunction